perf/benchmark/run_benchmark_job.sh

```diff
@@ -112,6 +109,10 @@ mkdir -p "${LOCAL_OUTPUT_DIR}"
 
 # Setup fortio and prometheus
 setup_metrics
+FORTIO_CLIENT_POD=$(kubectl get pods -n twopods | grep fortioclient | awk '{print $1}')
```

Comment:
could probably do similar to `kubectl get pods -n twopods -l app=fortioclient -oname` if you wanted